app/routes/task_routes.py

```diff
@@ -0,0 +1,128 @@
+from flask import Blueprint, jsonify, abort, make_response, request
+from app.models.task import Task
+from app import db
+import datetime
+import requests
+import os
+
+
+task_bp = Blueprint("tasks", __name__, url_prefix="/tasks")
+# Helper Function
+def validate_model(cls, model_id):
+    try:
+        model_id = int(model_id)
+    except:
+        abort(make_response({'message': f'Invalid {cls.__name__} id {model_id}'}, 400))
+
+    model = cls.query.get(model_id)
+    if not model:
+        print('in not found')
+        abort(make_response({'message': f'{cls.__name__} id {model_id} not found'}, 404))
+
+    return model
+
+def sort_by_title(tasks, order):
+    if order == 'asc':
+        return sorted(tasks, key=lambda task: task['title']) 
+    elif order == 'desc':
+        return sorted(tasks, key=lambda task: task['title'])[::-1]
+    else:
+        abort(make_response({'message': f"Sort {order} invalid.  Try 'asc' or 'desc'."}, 400))
+
```

Comment:
Very nice helper function here! I love how you are making use of the lambda functions!

Reply:
You could also have the database do it by implementing something like:
```python
  if sort_query == "asc":
        tasks = Task.query.order_by(Task.title.asc()).all()
    elif sort_query == "desc":
        tasks = Task.query.order_by(Task.title.desc()).all()
    else:
        tasks = Task.query.all()
```